src/molecule/data/molecule.json

```diff
@@ -0,0 +1,561 @@
+      "type": "object"
+    },
+    "ProvisionerConfigOptionsModel": {
+      "additionalProperties": true,
+      "properties": {
+        "defaults": {
+          "$ref": "#/$defs/ProvisionerConfigOptionsDefaultsModel"
+        },
+        "ssh_connection": {
+          "$ref": "#/$defs/ProvisionerConfigOptionsSshConnectionModel"
+        }
+      },
+      "title": "ProvisionerConfigOptionsModel",
+      "type": "object"
+    },
+    "ProvisionerConfigOptionsSshConnectionModel": {
+      "additionalProperties": false,
+      "properties": {
+        "control_path": {
+          "default": "%(directory)s/%%h-%%p-%%r",
+          "title": "Control Path",
+          "type": "string"
+        },
+        "scp_if_ssh": {
+          "default": true,
+          "title": "Scp If Ssh",
+          "type": "boolean"
+        }
+      },
+      "title": "ProvisionerConfigOptionsSshConnectionModel",
+      "type": "object"
+    },
+    "ProvisionerModel": {
+      "additionalProperties": true,
+      "properties": {
+        "config_options": {
+          "$ref": "#/$defs/ProvisionerConfigOptionsModel"
+        },
+        "env": {
+          "title": "Env",
+          "type": "object"
+        },
+        "inventory": {
+          "title": "Inventory",
+          "type": "object"
+        },
+        "log": {
+          "title": "Log",
+          "type": "boolean"
+        },
+        "name": {
+          "enum": ["ansible"],
+          "title": "Name",
+          "type": "string"
+        },
+        "playbooks": {
+          "title": "Playbooks",
+          "type": "object"
+        }
+      },
+      "title": "ProvisionerModel",
+      "type": "object"
+    },
+    "ScenarioSequence": {
+      "additionalProperties": false,
+      "items": {
+        "enum": [
+          "check",
+          "cleanup",
+          "converge",
+          "create",
+          "dependency",
+          "destroy",
+          "idempotence",
+          "lint",
+          "prepare",
+          "side_effect",
+          "syntax",
+          "test",
+          "verify"
+        ],
+        "type": "string"
+      },
+      "title": "ScenarioSequence",
+      "type": "array"
+    },
+    "VerifierModel": {
+      "additionalProperties": false,
+      "properties": {
+        "additional_files_or_dirs": {
+          "items": {
+            "type": "string"
+          },
+          "title": "AdditionalFilesOrDirs",
+          "type": "array"
+        },
+        "enabled": {
+          "title": "Enabled",
+          "type": "boolean"
+        },
+        "env": {
+          "title": "Env",
+          "type": "object"
+        },
+        "name": {
+          "default": "ansible",
+          "enum": ["ansible", "goss", "inspec", "testinfra"],
+          "title": "Name",
+          "type": "string"
+        },
+        "options": {
+          "title": "Options",
+          "type": "object"
+        }
+      },
+      "title": "VerifierModel",
+      "type": "object"
+    },
```

Comment:
The `VerifierModel` lacks the `directory` property, as noted in https://github.com/ansible-community/molecule/pull/3667. The last I looked, there was still code to process it, and it still appeared in documentation, so my assumption was the removal was an accident in the PR and it did not get caught in review.

Was this supposed to be removed? If so, could someone point me to the discussion about it and reasoning? It broke `testinfra` verification and I had to add it back on a fork to get unblocked. That would seem to indicate there is also a lack of tests to catch the regression. Could someone look into that?  Thanks!